src/appleseed/renderer/modeling/scene/scene.cpp

```diff
@@ -411,6 +405,12 @@ bool Scene::on_frame_begin(
     // Call on_frame_begin() on cameras last because some of them cast rays to sense depth in their autofocus mechanism.
     success = success && invoke_on_frame_begin(cameras(), project, this, recorder, abort_switch);
 
+    if (!Entity::on_frame_begin(project, parent, recorder, abort_switch))
```

Comment:
That doesn't look correct, the order of initialization was correct before and cannot be changed.

Reply:
Sorry, this shouldn’t be here. I forgot to remove it .